python/rmm/tests/test_rmm_pytorch.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+import rmm
+
+torch = pytest.importorskip("torch")
+
+
+@pytest.fixture(scope="session")
+def torch_allocator():
+    try:
+        from torch.cuda.memory import change_current_allocator
+    except ImportError:
+        pytest.skip("pytorch pluggable allocator not available")
+    change_current_allocator(rmm.rmm_torch_allocator)
```

Comment:
How does this function behave if passed None (the case where the torch allocator hasn't been defined)?

Reply:
Hmm - I wouldn't expect it to be `None` if we were able to import `change_current_allocator`, since the existence of `change_current_allocator` implies that `rmm_torch_allocator` was defined (although somewhat implicitly: `change_current_allocator` and `CudaPluggableAllocator` in PyTorch were introduced together).

Should we also skip this test if `rmm.rmm_torch_allocator` is `None` for some reason?

Reply:
Ah OK I wasn't sure about that, I thought they weren't introduced entirely concurrently. Up to you on the extra skip, it sounds like it would be pedantically correct but not practically necessary.